gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
Aha, OK. I don't remember the format anymore -- didn't realize there can be no strings at the end.

Anyway, can you split on the same chars as the original C? Since we're fixing the compatibility, we might as well fix it 100%. Cheers!